guides/v2.2/cloud/docker/docker-config.md

```diff
@@ -223,6 +223,9 @@ The `{{site.data.var.ct}}` version 2002.0.18 and later supports developer mode.
     ```bash
     bash ./mutagen.sh
     ```
+    
+    {: .bs-callout-info}
+    If you running docker on Windows and session start was failed, change `--symlink-mode` to `portable` in the `mutagen.sh` file.
 
     {: .bs-callout-info}
     If you use `docker-sync` for file synchronization, skip this step.
```

Comment:
```suggestion
```

Reply:
Recommend combining the two notes since they are in the same step. I know that @hguthrie is reworking the Docker content, so she might have some additional feedback on this PR.

Reply:
@oshmyheliuk  If you can make the following updates, then we can merge this PR.  Heather can incorporate additional updates in the PR that she's creating to improve  the Docker content.

Accept the suggestions to update the wording on the note you added and remove the second note.  Then, change line 221 to: 

```
1. Start the file synchronization with mutagen.io. If you use docker-sync for file synchronization, skip this step.
```